jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -941,14 +941,10 @@ pipeline {
             node(AGENT_LINUX_X64) {
                 checkout scm
                 script {
-                    closeBuildSuccessGithubIssue(
-                        message: buildMessage(search: 'Successfully built'),
-                        search: 'Successfully built',
-                        inputManifestPath: "manifests/$INPUT_MANIFEST"
-                    )
-                    if (params.CREATE_GITHUB_ISSUE) {
-                        createBuildFailureGithubIssue(
-                            message: buildMessage(search: 'Error building'),
+                    if (params.UPDATE_GITHUB_ISSUE) {
+                        UpdateBuildFailureIssues(
```

Comment:
Issue with camel case, should be `updateBuildFailureIssues`.

Reply:
Oh no! https://github.com/opensearch-project/opensearch-build-libraries/blob/main/vars/UpdateBuildFailureIssues.groovy

Will fix this in another iteration. Thanks for catching.